.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
     - TEST_SPECIFIC_MODULES=presto-raptor-legacy
     - TEST_SPECIFIC_MODULES=presto-accumulo
     - TEST_SPECIFIC_MODULES=presto-cassandra
-    - TEST_SPECIFIC_MODULES=presto-hive
+    - TEST_SPECIFIC_MODULES=presto-hive,presto-hive-hadoop2
```

Comment:
I'm not sure we want to bundle them together. Today, `presto-hive` takes 24 minutes and `presto-hive-hadoop2` takes 30 minutes. They are also fairly different types of tests.

Reply:
Together they took 54 minutes, see https://travis-ci.com/prestosql/presto/jobs/197565966. Which is ok, we may get back to it once it become a problem.

Reply:
Also notice that they are a single connector. Currently there is no reason to not to merge them. However in future (with hive3 and friends) we might have a reason to maintain this separation. So let's keep it as it is for now.

Reply:
Per [Travis docs](https://docs.travis-ci.com/user/customizing-the-build):

> When a job on a public repository takes longer than 50 minutes.

The fact that we aren't seeing failures until 60 minutes might be an undocumented grace period, or it could be a bug that may be fixed anytime. I don't think we should rely on the current behavior.

Also, having longer running jobs is problematic, as it sets the minimum latency for testing a PR. If we have two jobs of 30 minutes each, we can know after 30 minutes if it passed. With jobs taking nearly an hour, we have to wait much longer.

Reply:
I feel like we are merging them here because it is convenient for the way the new test code is structured, not because it is the right thing to do.

Reply:
Travis doesn't kill jobs at 50th minute because we pay Travis (patience is gratis).
However, Travis kills jobs in personal forks. I get travis failure emails from findepi/presto often (i know, i can filter out the mail; that's not the point). We should aim at having tests pass in personal forks too.